flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
why?

Reply:
Why is this wrong?

Reply:
why only support this architecture?

Reply:
I could build for both but I can only test for x86_64. In hindsight, I don't really see any reason why the app shouldn't work on arm. I could remove this.